.gitignore

```diff
@@ -18,3 +18,4 @@ packages/@okta/courage-dist/*.html
 testenv
 packages/@okta/i18n/src/json/
 packages/@okta/i18n/src/properties/*_in.*
+packages/@okta/i18n/dist
```

Comment:
Should this be added to .gitignore? If so I should add it in my PR: https://github.com/okta/okta-signin-widget/pull/1092

Reply:
I confirmed with santhosh this should be in the .gitignore. I have made the same change in PR 1092